config/action.d/pbr.conf

```diff
@@ -0,0 +1,32 @@
+# Fail2Ban configuration file
+#
+# Author: Konstantin Shalygin
+#
+# Policy Based Routing rules
+#
+# Provides the same features, like 'route' action, and fixes the main issue of
+# this hardening blocking: block per ip + service port + service protocol.
+# Actually you can add more options and do more granular blocking, see `ip rule
+# add help`
+
+[Definition]
+actionstart = ip route add unreachable 0.0.0.0/0 table <tableid>
```

Comment:
This is a difference to route action, where "unreachable" can be controlled by `<blocktype>`.

So maybe better something like that:
```diff
- actionstart = ip route add unreachable 0.0.0.0/0 table <tableid>
+ actionstart = ip route add <blocktype> 0.0.0.0/0 table <tableid>
- actionstop = ip route delete 0.0.0.0/0 table <tableid>
+ actionstop = ip route delete <blocktype> 0.0.0.0/0 table <tableid>
```